### DISCUSSION TITLE
matchUpdateTypes rules not being followed

### DISCUSSION THREAD

**Discussion by simao-silva** [renovatebot/renovate#13722] [Request Help]
I am using a sefl hosted version of Renovate 31.38.4 and I have the following configuration: 
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":disableDependencyDashboard"
  ],
  "separateMinorPatch": true,
  "enabledManagers": ["pip_requirements", "dockerfile", "droneci"],
  "packageRules": [
    {
      "enabled": true,
      "matchManagers": ["dockerfile"],
      "matchUpdateTypes": ["patch"]
    },
    {
      "enabled": true,
      "matchManagers": ["pip_requirements", "droneci"],
      "matchUpdateTypes": ["major", "minor", "patch"]
    }
  ]
}
```
What I want is to only update the base image in the Dockerfile to patch versions, for example from `python:3.9.9` to `python:3.9.10`. However, Renovate keeps opening PRs for a minor update to version `3.10.2`. How should I configure renovate.json so that only patch updates are opened?

**Comment by rarkins** [comment]
You need to flip your logic around.

By default EVERYTHING is enabled. Having a package rule which sets `enabled=true` does nothing unless you had a previous one which set things to `false`. So do something like this instead:

```
    {
      "matchManagers": ["dockerfile"],
      "matchUpdateTypes": ["minor", "major"],
      "enabled": false
    },
```

**Reply by simao-silva** [reply]
Thank you. That fixed my issue. Still, what is the right configuration for Renovate to update Python images using tags format like `3.9.10-alpine3.14 -> 3.9.10-alpine3.15`?

**Reply by rarkins** [reply]
There's nothing out of the box, but it's probably achievable using regex manager
